### ISSUE TITLE
ModuleNotFoundError: No module named 'transformers.hf_api'

### ISSUE THREAD

**Issue by edanweis** [neuml/txtai#112]
With no changes to my code, Streamlit is now failing with:

```Python
2021-09-02 05:17:40.602 Loading faiss.
2021-09-02 05:17:40.620 Successfully loaded faiss.
2021-09-02 05:17:43.062 Uncaught app exception
Traceback (most recent call last):
  File "/home/appuser/venv/lib/python3.9/site-packages/streamlit/script_runner.py", line 350, in _run_script
    exec(code, module.__dict__)
  File "/app/km-streamlit/streamlit_app.py", line 8, in <module>
    from txtai.embeddings import Embeddings
  File "/home/appuser/venv/lib/python3.9/site-packages/txtai/embeddings/__init__.py", line 5, in <module>
    from .base import Embeddings
  File "/home/appuser/venv/lib/python3.9/site-packages/txtai/embeddings/base.py", line 12, in <module>
    from ..scoring import ScoringFactory
  File "/home/appuser/venv/lib/python3.9/site-packages/txtai/scoring/__init__.py", line 5, in <module>
    from .base import Scoring
  File "/home/appuser/venv/lib/python3.9/site-packages/txtai/scoring/base.py", line 10, in <module>
    from ..pipeline.tokenizer import Tokenizer
  File "/home/appuser/venv/lib/python3.9/site-packages/txtai/pipeline/__init__.py", line 20, in <module>
    from .translation import Translation
  File "/home/appuser/venv/lib/python3.9/site-packages/txtai/pipeline/translation.py", line 15, in <module>
    from transformers.hf_api import HfApi
    ModuleNotFoundError: No module named 'transformers.hf_api'
```

Could this be a HF breaking change?

**Comment by davidmezzetti** [comment]
transformers 4.10.0 introduced a couple breaking changes to txtai. There is a fix in the master branch for this (#110) and will be pushed with the next release.

In the meantime, if you force transformers==4.9.2

```
pip install transformers==4.9.2
```

This can also be done when you install txtai

```
pip install txtai transformers==4.9.2
```

**Comment by davidmezzetti** [comment]
Fixed in v3.3.0
